init.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE clients (
     name VARCHAR(255) NOT NULL, 
     created_at TIMESTAMP, 
     updated_at TIMESTAMP, 
-    email VARCHAR(255) UNIQUE 
+    email VARCHAR(255)
```

Comment:
why this is changed?)

Reply:
I mistakenly removed the UNIQUE constraint. Email must be unique)